mon-pix/app/pods/components/module/qrocm/styles.scss

```diff
@@ -0,0 +1,30 @@
+.element-qrocm {
+  @extend %pix-body-l;
+
+  padding: $pix-spacing-s;
+
+  // revert the reset
+  ul:not([role=listbox]),
+  ol {
+    margin: revert;
+    padding: revert;
+    list-style: revert;
+  }
```

Comment:
@yannbertrand ma solution un peu dégueu pour cibler tous les `ul` sauf celui de pix-ui :( (à challenger de ouf évidemment)

Reply:
J'ai proposé `ul:not([class])` qui fonctionne :) toujours pas parfait, je sais pas trop comment faire mieux